spinalcordtoolbox/registration/landmarks.py

```diff
@@ -41,7 +41,8 @@
 ini_param_trans_y = -150.0  # pix
 initial_step = 2
 
-def register_landmarks(fname_src, fname_dest, dof, fname_affine='affine.txt', verbose=1, path_qc=None):
+
+def register_landmarks(fname_src, fname_dest, dof, fname_affine='affine.txt', verbose=1):
```

Comment:
What is the motivation for the changes in `landmarks.py`? 

(I'm not necessarily opposed, they are just unexpected to me given the scope of this PR. I am wondering why they were included when they appear to be unrelated.)

Reply:
> is it just because path_qc was unused by calls to register_landmarks?

indeed that's it-- indeed, out of scope, i just did it to save time